### ISSUE TITLE
register PkgCleanup?

### ISSUE THREAD

**Issue by StefanKarpinski** [giordano/PkgCleanup.jl#7]
Would be nice to be able to just do `using PkgCleanup` in the REPL and get it auto-installed.

**Comment by giordano** [comment]
Yeah, I probably should, but I feel bad publishing a package with this single test https://github.com/giordano/PkgCleanup.jl/blob/0cfb858bb7253c00fb6c357ea9f6a678858b2276/test/runtests.jl#L5 :cry:

**Comment by StefanKarpinski** [comment]
That's a fair concern!

**Comment by hdavid16** [comment]
Why not add the functionality of this package to Pkg.jl?

**Comment by StefanKarpinski** [comment]
All the usual reasons not to add things to Julia that can be external:

1. It inflates the size of the Julia download and binary
2. It locks this package into the Pkg and Julia lifecycle, which is not fun as a developer
3. It disallows making any breaking API changes, if the current API is ever deemed suboptimal
4. Adds more development burden to Julia/Pkg in terms of maintenance, testing and surface area
5. There's no compelling reason to since we have a fast, reliable package manager

**Comment by StefanKarpinski** [comment]
Bump. I know there aren't a lot of tests, but how about registering this and if someone wants to add tests they can. It's primarily an interactive tool anyway, so it's a bit of a different situation. Speaking of which, maybe a use case for the new app functionality?

**Comment by giordano** [comment]
Ok, let's do it.  Before that, I'm happy to move this to an org, do you have in mind any org which be a good match?

**Comment by StefanKarpinski** [comment]
We don't seem to actually have a good org for this kind of package. Maybe we should create a JuliaPkgTools org or something like that?
